src/app/qgsrelationadddlg.cpp

```diff
@@ -35,7 +35,7 @@ QgsRelationAddDlg::QgsRelationAddDlg( QWidget *parent )
 
   mCbxRelationStrength->addItem( "Association", QVariant::fromValue( QgsRelation::RelationStrength::Association ) );
   mCbxRelationStrength->addItem( "Composition", QVariant::fromValue( QgsRelation::RelationStrength::Composition ) );
-  mCbxRelationStrength->setToolTip( QStringLiteral( "On composition, the child features will be duplicated too. Duplications are made by the feature duplication default action" ) );
+  mCbxRelationStrength->setToolTip( QStringLiteral( "On composition, the child features will be duplicated too.\nDuplications are made by the feature duplication action.\nThe default actions are activated in the Action section of the layer properties." ) );
```

Comment:
I find the terminology around relations to be quite technical and difficult to consume. I wonder if we can't come up with some more simple, descriptive language that really describes the implications of these choices in a way that an average user might understand?

Reply:
I think the reason behind that is that on the one hand, the concept itself is quite wide (e.g. for compositions the term children is appropriate, for associations not really), on the other hand there's a certain limitation in language skills (looking at myself here) that makes it hard. ain short, I agree, but find it hard to come up with something better.

Reply:
@signedav FYI, tooltips support some basic HTML (`<b>Title</b>`, `<p>Paragraph</p>`, ...)